tests/test_infosets.py

```diff
@@ -52,3 +60,47 @@ def test_infoset_add_action_error():
     game = games.read_from_file("basic_extensive_game.efg")
     with pytest.raises(gbt.MismatchError):
         game.add_action(game.players[0].infosets[0], game.players[1].infosets[0].actions[0])
+
+def test_append_move_single_infoset_ist_of_nodes():
+    """Test that appending a list of nodes creates a single infoset."""
+    game = _create_game()
+    nodes = [game.root.children[1],
+             game.root.children[0].children[0],
+             game.root.children[0].children[1]]
+    game.append_move(nodes, "Player 3", ["B", "F"])
+
+    assert len(game.players[2].infosets) == 1
+
+
+def test_append_move_infoset_members_list_of_nodes():
+    """Test that the newly created infoset from a list of nodes is inherited
+    with as many members as the cardinality of the list.
```

Comment:
Would not a better test be to confirm that the correct nodes are in fact in the same information set, rather than just the length is correct?